_includes/footer.html

```diff
@@ -22,6 +22,18 @@
         <span class="username">{{ site.github_username }}</span>
       </a>
     </li>
+       <li>
```

Comment:
There needs to be a `{% endif %} {% if site.mastodon_username %}` before this line to show a mastodon icon only if a mastodon username is configured.